src/components/PetitionWidget/index.js

```diff
@@ -2,21 +2,17 @@ import React from 'react';
 import styles from './petition-widget.scss';
 import Container from 'components/Container';
 import Section from 'components/Section';
+import WidgetBranding from 'components/WidgetBranding';
 import WidgetSupportButton from 'containers/widget/WidgetSupportButton';
 import PetitionWidgetHeader from 'containers/widget/PetitionWidgetHeader';
-import PetitionFooter from 'containers/PetitionFooter';
 
 const PetitionWidget = () => (
   <article className={styles.root}>
     <Section>
       <Container>
         <PetitionWidgetHeader />
         <WidgetSupportButton />
```

Comment:
re: naming, could it be `<PetitionWidgetSupportButton>` as it's petition-specific like the include above?